interfaces/mount/spec.go

```diff
@@ -39,10 +46,109 @@ func (spec *Specification) AddMountEntry(e Entry) error {
 	return nil
 }
 
+// resolveSpecialVariable resolves one of the three $SNAP* variables at the
+// beginning of a given path.  The variables are $SNAP, $SNAP_DATA and
+// $SNAP_COMMON. If there are no variables then $SNAP is implicitly assumed
+// (this is the behavior that was used before the variables were supporter).
+func resolveSpecialVariable(path string, snapInfo *snap.Info) string {
```

Comment:
Would it make sense to use os.Expand() here?

Reply:
Oh *indeed* this is much nicer. I will use it!

Reply:
Updated! I will probably go and use it in a few more places now :)